RNExample/android/app/proguard-rules.pro

```diff
@@ -0,0 +1,10 @@
+# Add project specific ProGuard rules here.
```

Comment:
Is it possible to remove the Android part of this example project entirely? I know by default it creates both, but it should be possible to make an iOS only RN project.